### ISSUE TITLE
Wrong argument type for figsize in documentation for figure

### ISSUE THREAD

**Issue by monocongo** [matplotlib/matplotlib#13208]
### Bug report

**Bug summary**

The documentation for [matplotlib.pyplot.figure](https://matplotlib.org/api/_as_gen/matplotlib.pyplot.figure.html#matplotlib.pyplot.figure) shows the wrong type for the figsize argument's tuple -- it is float but instead the docs say integer.

**Actual outcome**

```
figsize : tuple of integers, optional, default: None
width, height in inches. If not provided, defaults to rcParams["figure.figsize"] = [6.4, 4.8].
```

**Expected outcome**

```
figsize : tuple of floats, optional, default: None
width, height in inches. If not provided, defaults to rcParams["figure.figsize"] = [6.4, 4.8].
```

**Comment by tacaswell** [comment]
Indeed, the docstring is wrong!  Can you put in a pull-request with that change?

**Comment by QuLogic** [comment]
Fixed by #13212.

If you write Fixes #13208 (or Resolves or Closes or maybe a few others), it will automatically close this issue when the PR is merged.
